cluster/examples/kubernetes/monitoring/prometheus.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: prometheus
   namespace: rook
 ---
-apiVersion: rbac.authorization.k8s.io/v1alpha1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Shouldn't this be `v1` for 1.8?

Reply:
yes but i am not sure if `v1` works for K8s 1.7.x or 1,6.x. I tried `v1beta1` on 1.8 and it worked. I can give it a try later.